src/mixins/stripe.js

```diff
@@ -88,9 +88,10 @@ var StripeMixin = {
       return;
     }
     if (!window.grecaptcha) {
-      return setTimeout(() => {
+      setTimeout(() => {
         this.expectRecaptcha(callback);
       }, 100);
+      return;
```

Comment:
Why this change? It doesn't appear to do anything

Reply:
Strictly not necessary, it doesn't change the behaviour. But it's not good practice to have methods that sometimes return something (well undefined, so not much) and sometimes not, as some linters will tell you...